omero/sysadmins/server-upgrade.txt

```diff
@@ -218,8 +218,8 @@ Run the upgrade script
 """"""""""""""""""""""
 
 You **must** use the same username and password you have defined during
-:doc:`unix/server-installation`. The |version| upgrade script should execute
-in a short time.
+:doc:`unix/server-installation`. For a large production system you
+should plan for the upgrade script to take hours to run.
```

Comment:
That wording would work for me - "For a large production system you should plan for the fact that the upgrade script may take several hours to run." ?